src/server/errorHandler.js

```diff
@@ -1,17 +1,35 @@
+const logger = require('../logger')
 
-const logErrors = () => {
-  return (error, req, res) => {
+/* eslint-disable */
+const errorHandler = () => {
+  return (err, req, res, next) => {
+    let errorCode = err.statusCode || 500
+    let errorMessage = err.message || 'General error.'
+    if (err.isBoom) {
+      errorCode = err.output.statusCode
+      errorMessage = err.output.payload.message
+    }
 
-    let errorCode = error.statusCode || 500
-    let errorMessage = error.message || 'General error.'
-  
-    console.log(`${errorCode} - ${errorMessage}`)
-    if (error.stack) {
-      console.log('##### Error Stack: ', error.stack)
+    let response = { message: `${errorCode} - ${errorMessage}` }
```

Comment:
In a prod environment we should update `errorMessage = 'Internal server error'` before this point to avoid unnecessarily interpolating this string twice (once with the original error message that won't be sent in prod)

Reply:
Should do that a bit further down

Reply:
Right but in prod it'll first assign this var by interpolating this string, then replace it with another interpolated string, which is wasted computation. Only need to assign `response` once